mobile/Screens/WritingTipScreen.jsx

```diff
@@ -0,0 +1,44 @@
+import { React, useState, useEffect } from 'react';
+import {
+  StyleSheet, Text, View, Button,
+} from 'react-native';
+import axios from 'axios';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+});
+
+export default function WritingTipScreen() {
+  const [writingTip, setWritingTip] = useState('');
+  const handleGetWritingTip = async () => {
+    try {
+      const randomWritingTip = await axios.get(`${process.env.EXPO_PUBLIC_SERVER_URL}/writingTip/get`, { timeout: 20000 });
+      setWritingTip(randomWritingTip.data);
+      return randomWritingTip.data;
+    } catch (err) {
+      console.log(err);
+    }
+    return 'True';
+  };
+
+  useEffect(() => {
+    // handleGetWritingTip(); unsure why we need this
```

Comment:
This useEffect would display a writing tip as soon as the screen is rendered. But since the designers are still working on writing tip frontend styles/deciding if they want to display on render, it is ok to keep this commented for now!